### PR TITLE
Add java requirement to fasta_nucleotide_color_plot

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <tool id="fasta_nucleotide_color_plot" name="Fasta nucleotide color plot" version="1.0.0">
     <description></description>
+    <requirements>
+        <requirement type="package" version="8.0.92">java-jdk</requirement>
```
Comment (bgruening) [inline tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml]: Can you use the conda-forge package? it should be names openjdk I think. BioConda will soon move to this one.

Comment (gregvonkuster) [inline tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml]: Sure!  Sorry, I think you pinged me on this recently, but I didn't get it at the time.  Just a sec, I'll update my PRs.

**Comment by gregvonkuster** [comment]
@bgruening should I merge all of my PRs that have passed tests?

**Comment by bgruening** [comment]
Sorry Greg, was busy with funding stuff the entire day.

**Comment by gregvonkuster** [comment]
No need to apologize!  No one is as busy as you!  ;)

**Pull request merged**